crates/swc_ecma_minifier/tests/fixture/next/styled-components/1/output.js

```diff
@@ -5834,9 +5833,9 @@
                 return function(props) {
                     void 0 === props && (props = {});
                     var styles, theme1 = _extends({}, defaultTheme, {}, props.theme || props), result = {}, styles1 = (styles = "function" == typeof args ? args(theme1) : args, function(theme) {
-                        var next = {}, breakpoints = index_esm_get(theme, "breakpoints", defaultBreakpoints), mediaQueries = [
+                        var next = {}, mediaQueries = [
```

Comment:
`breakpoints` is inlined as `[null].concat` (property access) cannot modify `breakpoints`